google-maps/android/build.gradle

```
@@ -22,7 +22,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.2.1'
+        classpath 'com.android.tools.build:gradle:7.4.2'
```

```suggestion
        classpath 'com.android.tools.build:gradle:7.4.1'
```
for now we are using 7.4.1 since that's what latest Android Studio recommends